lang/ruby/lib/avro/schema_validator.rb

```diff
@@ -183,6 +187,10 @@ def ruby_to_avro_type(ruby_class)
           Hash => 'record'
```

Comment:
If a Ruby prior to 2.4 is used, won't the `Fixnum` and `Bignum` keys still be needed? Prior to Ruby 2.4, values will still return those as their class and they will be passed to this method.